sdk/ml/azure-ai-ml/pyproject.toml

```
@@ -3,3 +3,12 @@ mypy = false
 pyright = false
 type_check_samples = false
 verifytypes = false
+
+[tool.interrogate]
+fail-under = 95
```

```suggestion
fail-under = 35
```

potential bug?  It was initially 35 when it was listed as an argument